src/Entity/ShipFrom.php

```diff
@@ -21,8 +21,8 @@ public function toNode(DOMDocument $document = null)
 
         $node = $document->createElement('ShipFrom');
 
-        if ($this->getCompanyName()) {
-            $node->appendChild($document->createElement('CompanyName', $this->getCompanyName()));
+        if ($this->getCompanyDisplayableName()) {
```

Comment:
I still think this should be $this->getCompanyName(). I don't see any use introducing getCompanyDisplayableName while getCompanyName did exactly the same. Also the XML tag is CompanyName, so it can get confusing.